Java/Fold.tmPreferences

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+    <key>scope</key>
+    <string>source.java</string>
+    <key>settings</key>
+    <dict>
+        <key>foldScopes</key>
+        <array>
+            <dict>
+                <key>begin</key>
+                <string>punctuation.section.block.begin</string>
+                <key>end</key>
+                <string>punctuation.section.block.end</string>
+            </dict>
```

Comment:
Folding groups should be safe as well in Java.

```suggestion
            <dict>
                <key>begin</key>
                <string>punctuation.section.block.begin</string>
                <key>end</key>
                <string>punctuation.section.block.end</string>
            </dict>
            <dict>
                <key>begin</key>
                <string>punctuation.section.group.begin</string>
                <key>end</key>
                <string>punctuation.section.group.end</string>
            </dict>            
```

Reply:
Unfortunately not, see for example syntax_test_java.java:6137:

```
    for (; {}
//  ^^^^^^^^^ - meta.statement.loop.for meta.statement.loop.for
//  ^^^^ meta.statement.loop.for.java - meta.group
//      ^^^ meta.statement.loop.for.java meta.group.java
//         ^^ meta.statement.loop.for.java meta.block.java
//  ^^^ keyword.control.loop.for.java
//     ^ - keyword
//      ^ punctuation.section.group.begin.java
//       ^ punctuation.terminator.java
//         ^ punctuation.section.block.begin.java
//          ^ punctuation.section.block.end.java
```